assets/webpack.config.js

```diff
@@ -287,10 +307,19 @@ module.exports = (env, options) => ({
     ],
   },
   plugins: [
+    BUNDLE_ANALYZER_ENABLED ? new BundleAnalyzerPlugin() : undefined,
     new MiniCssExtractPlugin({
       filename: '../css/[name].css',
     }),
-    new CopyWebpackPlugin({ patterns: [{ from: 'static/', to: '../' }] }),
+    new CopyWebpackPlugin({
+      patterns: [
+        {
+          from: 'static/',
+          to: '../',
+          filter: async (path) => path.indexOf('.map') == -1, // these were causing a duplicate file error on a production build
```

Comment:
Once minimization was actually running, webpack would fail here, so filtering out the .map files.